File: backend/core-api/src/main/java/ai/govbiz/core/client/bizinfo/BizInfoProgramPayload.java
package ai.govbiz.core.client.bizinfo;

public record BizInfoProgramPayload(
        String title,
        String sourceUrl,
        String id,
        String jurisdictionOrganization,
        String executingOrganization,
        String summaryHtml,
        String category,
        String createdAt,
        String applicationPeriod,
        String updatedAt,
        String target,
        String hashtags,
        String applicationMethod,
        String applicationUrl
) {
}

File: backend/core-api/src/main/kotlin/ai/govbiz/core/supportprogram/dto/bizinfo/BizInfoProgramPayload.kt
package ai.govbiz.core.supportprogram.dto.bizinfo

data class BizInfoProgramPayload(
    val title: String?,
    val sourceUrl: String?,
    val id: String?,
    val jurisdictionOrganization: String?,
    val executingOrganization: String?,
    val summaryHtml: String?,
    val category: String?,
    val createdAt: String?,
    val applicationPeriod: String?,
    val updatedAt: String?,
    val target: String?,
    val hashtags: String?,
    val applicationMethod: String?,
)
